utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -188,6 +196,20 @@ object SwaggerTyped {
       TypedObjectTypingResult(elementType.mapValuesNow(typingResult).toList.sortBy { case (propertyName, _) => propertyName })
     }
   }
+
+  private def inferredIntType(minValue: BigDecimal, exclusiveMinValue: BigDecimal, maxValue: BigDecimal, exclusiveMaxValue: BigDecimal): SwaggerTyped = {
+
+    val lowerBoundary: Option[BigDecimal] = List(Option(exclusiveMinValue).map(_ + 1), Option(minValue)).flatten.sorted(Ordering.BigDecimal.reverse).headOption
+    val upperBoundary: Option[BigDecimal] = List(Option(exclusiveMaxValue).map(_ - 1), Option(maxValue)).flatten.sorted(Ordering.BigDecimal).headOption
+
+    List(lowerBoundary, upperBoundary).flatten match {
+      case min :: max :: Nil if min.isValidInt && max.isValidInt => SwaggerInteger
+      case min :: max :: Nil if min.isValidLong && max.isValidLong => SwaggerLong
+      case _ :: _ :: Nil => SwaggerBigInteger
+      case value :: Nil if !value.isValidLong => SwaggerBigInteger
+      case _ => SwaggerLong
```

Comment:
I think that we should add a comment here, that in this case it potentially be a big integer, but because it is default, we don't want users to work with painful big integers. WDYT?

Reply:
comment added

Reply:
nice, thx